--- cuetools-1.3.1/extras/cuetag.sh.orig	2008-08-31 18:50:09.000000000 +0200
+++ cuetools-1.3.1/extras/cuetag.sh	2008-08-31 18:51:19.000000000 +0200
@@ -18,7 +18,7 @@ vorbis()
 {
 	# FLAC tagging
 	# --remove-vc-all overwrites existing comments
-	METAFLAC="metaflac --remove-vc-all --import-vc-from=-"
+	METAFLAC="metaflac --remove-all-tags --import-tags-from=-"
 
 	# Ogg Vorbis tagging
 	# -w overwrites existing comments
